plugins/engines/ospray/render/utils/AbstractRenderer.ih

```diff
@@ -141,3 +157,14 @@ float shadedLightIntensity(
     const varying vec3f& normal,
     varying vec3f& lightDirection );
 
+
+/**
+    Returns the color and opacity of a ray going through the volume attached to the scene
+    @param self Pointer to the current renderer
+    @param ray Current ray used to traverse the volume
+    @return Resulting color and alpha value
+*/
+vec4f getVolumeValue(
```

Comment:
how about naming it getVolumeColor ? I didn't immediately understand in the code above that this was returning the color, I though it was the value of a single voxel